src/control/dmg/pool.go

```diff
@@ -58,6 +59,49 @@ type CreatePoolCmd struct {
 	Sys        string `short:"S" long:"sys" default:"daos_server" description:"DAOS system that pool is to be a part of"`
 }
 
+// Execute is run when CreatePoolCmd subcommand is activated
+func (c *CreatePoolCmd) Execute(args []string) error {
+	// broadcast == false to connect to mgmt svc access point
+	if err := appSetup(false); err != nil {
+		return err
+	}
+
+	if err := createPool(
+		c.ScmSize, c.NVMeSize, c.RankList, c.NumSvcReps,
+		c.GroupName, c.UserName, c.Sys, c.ACLFile); err != nil {
+
+		return err
+	}
+
+	// exit immediately to avoid continuation of main
+	os.Exit(0)
+	// never reached
+	return nil
+}
+
+// DestroyPoolCmd is the struct representing the command to destroy a DAOS pool.
+type DestroyPoolCmd struct {
+	Uuid  string `short:"u" long:"uuid" required:"1" description:"UUID of DAOS pool to destroy"`
+	Force bool   `short:"f" long:"force" description:"Force removal of DAOS pool"`
+}
+
+// Execute is run when DestroyPoolCmd subcommand is activated
+func (d *DestroyPoolCmd) Execute(args []string) error {
+	// broadcast == false to connect to mgmt svc access point
+	if err := appSetup(false); err != nil {
+		return err
+	}
+
+	if err := destroyPool(d.Uuid, d.Force); err != nil {
```

Comment:
Should there be some validation on uuid here before sending it on? Checking to make sure it's not an empty string, that it is formatted like a uuid, etc. Presumably the server is validating too but it might be better to catch some errors quickly.

Reply:
it is a "required" Field so an empty string won't be accepted